src/parser/parseTransformAttribute.ts

```diff
@@ -110,13 +55,14 @@ export function parseTransformAttribute(attributeValue) {
     return matrix;
   }
 
-  attributeValue.replace(reTransform, function (match) {
-    const m = new RegExp(transform).exec(match).filter(function (match) {
-        // match !== '' && match != null
-        return !!match;
-      }),
-      operation = m[1],
-      args = m.slice(2).map(parseFloat);
+  for (const match of attributeValue.matchAll(reTransform)) {
+    const transformMatch = new RegExp(transform).exec(match[0]);
+    if (!transformMatch) {
+      continue;
+    }
+    const matchedParams = transformMatch.filter((m) => !!m);
+    const operation = matchedParams[1];
+    const args = matchedParams.slice(2).map(parseFloat);
```

Comment:
```suggestion
    const [ _, operation, ...rawArgs ] = matchedParams;
```
Would this work?

then

```js
const args = rawArgs.map((arg) => parseFloat(arg));
```

There is a risk using parseFloat inside a map like that, that we are calling parseFloat as


```js
(value, index, wholeArray) => parseFloat(value, index, wholeArray)
```

And you never know what happens

Reply:
I agree
Risky  
Should not abbreviate